providers/base/units/stress/suspend_cycles_reboot.pxu

```diff
@@ -0,0 +1,179 @@
+id: stress_s3_cycles_iterations_1
```

Comment:
You should add a few lines of comments at the beginning of the file with:

- a short explanation of what this file is about
- a mention of the MD file for people willing to know more